archaius2-archaius1-bridge/src/main/java/com/netflix/archaius/bridge/StaticAbstractConfiguration.java

```diff
@@ -80,8 +80,11 @@ public boolean containsKey(String key) {
 
     @Override
     public String getString(String key, String defaultValue) {
-        Object value = getProperty(key);
-        return value != null ? value.toString() : defaultValue;
+        if (delegate == null) {
```

Comment:
The fix for getting non-string types as string has been pushed down to the archaius Config so it can do the interpolation